tests_common/pytest_plugin.py

```diff
@@ -1712,3 +1716,15 @@ def create_db_api_hook(request):
     test_db_hook.escape_column_names = escape_column_names or False
 
     return test_db_hook
+
+
+@pytest.fixture(autouse=True, scope="session")
```

Comment:
This is how we make sure that "tests" folders are root you can import from "from integration.apache.beam" ....